widgets/PySparkProgram-sparkprogram.json

```diff
@@ -0,0 +1,30 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
```

Comment:
i think this is supposed to be some higher version (1.4? don't remember). Unless the UI supports new features (icon) even with an older spec version?